src/insider-threats/mitigating-dprk-it-workers.md

```diff
@@ -0,0 +1,88 @@
+9.  Set up a final confirmation call after all access is revoked and you are certain your organization is safe from any insider threats. **Attempt to perform KYC under some pretense. Do not settle for document scans, as these are easily obtainable.** Insist on a "video call" KYC, where the actor must show the physical document they previously provided next to their face.
+10. Attempt to geolocate them. Send a document that tracks the IP address used to open it. Look for VPN usage or IP addresses in the DPRK/Eastern Russian range.
+11. Collect all on-chain/payroll data on funds transferred to the actor. If an actor is confirmed as DPRK-related, you will be required to file a Suspicious Activity Report (SAR). We would appreciate you sharing this data with us later for public security reasons and future investigations. The data will remain private.
+12. You do not need to inform the DPRK IT Worker about the actual reasons for termination. You can cite downsizing, poor performance, lack of cultural fit, or any other reason.
+13. **In the post-mortem phase, you will need to halt all development on the affected infrastructure.** For companies that hired a DPRK IT Worker as a "Core Engineer," this may sometimes mean several weeks of re-auditing all code and infrastructure. **You should stop and do it.** At the same time, build a threat model to prevent the situation from recurring and to limit its effects if it does happen again. You can remove the DPRK IT Worker's code from your codebase. **DPRK IT Workers will claim this code as part of their work experience in their future job searches.** The decision is ultimately yours. While not removing the code helps security researchers like those at SEAL911 track these workers, our recommendation is to remove it to protect your project and deny adding credibility to DPRK IT Worker.
+
+#### Data collection
+1.  You should collect the following data before firing the DPRK IT Worker. This data will help organizations like SEAL with future mitigation and the discovery of other actors.
+    1.  Full Legal Name
+    2.  Phone
+    3.  Email(s)
+    4.  Location (City/State)
+    5.  LinkedIn
+    6.  Telegram/Discord
+    7.  GitHub
+    8.  Copy of Resume
+    9.  Date Hired
+    10. Sourcing (Where were they hired from? Telegram, Staffing, Recruiter, etc.)
+    11. Company Email (Yes/No)
+    12. IP addresses (from email or any other sources at your disposal)
+    13. Managed Laptop (Yes/No)
+        1.  Shipment address (for the laptop or other items)
+    14. KYC data (ID/Passport/Bills document scans)
+    15. Photos/Screenshots (physical appearance, potentially from a video call)
+    16. Payment data **(Extremely important!)**
+        1.  On-chain addresses provided for payment
+        2.  Bank data
+    17. Did they refer anyone else?
+    18. Were they referred by someone else?
+
+#### Overview of risks to your organization
```

Comment:
Suggest some others to add:
- Supply-chain compromise - DPRK IT Workers may intentionally introduce vulnerabilities that impact down-stream projects that depend on your software / services (e.g. SafeWallet UI in the ByBit hack)
- Reputational damage to your brand and loss of trust of your users and customers
- Asset freeze / loss of access to financial services: your assets may be frozen or seized, and financial institutions (e.g. banks, exchanges) may terminate your access if you are suspected of funding sanctioned entities
- Criminal investigations - law enforcement may investigate your involvement and impose fines or press criminal charges against your organization